pkg/crds/controllers/batch/batchjob_controller_helpers.go

```diff
@@ -46,6 +46,7 @@ import (
 	kbatch "k8s.io/api/batch/v1"
 	kcore "k8s.io/api/core/v1"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
nit: consider calling it `kmeta` or `metav1` or something similar (it seems like we have it as `kmeta` in our other files)

Reply:
let's stick with `kmeta` 👍